pkg/bindings/pods/pods.go

```diff
@@ -264,3 +260,26 @@ func Unpause(ctx context.Context, nameOrID string) (*entities.PodUnpauseReport,
 	}
 	return &report, response.Process(&report)
 }
+
+// Stats display resource-usage statistics of one or more pods.
+func Stats(ctx context.Context, namesOrIDs []string, options entities.PodStatsOptions) ([]*entities.PodStatsReport, error) {
+	if options.Latest {
+		return nil, errors.New("latest is not supported")
+	}
```

Comment:
Down below it looks like latest is supported?  https://github.com/containers/libpod/pull/5966/files#diff-839acaa99d80ae77f5c7ba85b158c719R224

Reply:
That's for allowing code share between the local and remote client. As they both implement the same interface, we have to make a check here.